src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -344,6 +344,22 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
                 return Promise.resolve();
             }));
 
+        if (await this.driver.isGeneratedColumnsSupported(this.mode)) {
+            // if table have column with generated type, we must add the expression to the meta table
+            await Promise.all(table.columns
+                .filter(column => column.generatedType === "STORED" && column.asExpression)
+                .map(async column => {
+                    const tableName = await this.getTableNameWithSchema(table.name);
+                    const deleteQuery = new Query(`DELETE FROM typeorm_generation_meta WHERE table_name = $1 AND column_name = $2`, [tableName, column.name]);
+                    upQueries.push(deleteQuery);
+                    upQueries.push(new Query(`INSERT INTO typeorm_generation_meta(table_name, column_name, generation_expression) VALUES ($1, $2, $3)`, [tableName, column.name, column.asExpression]));
+                    downQueries.push(deleteQuery);
+                    return Promise.resolve();
```

Comment:
Do you actually need to return `Promise.resolve()` here?

Reply:
Not really. I will remove it.